NEWS.md

```diff
@@ -136,6 +136,9 @@ Language changes
     backslashes and the end of the literal while 2n+1 backslashes followed by a quote encodes n
     backslashes followed by a quote character ([#22926]).
 
+  * `readdir()` output is now guaranteed to be sorted on any OS. A new `sort` keyword argument can
+    be used to disable explicit sorting ([#24626]).
```

Comment:
This isn't a language change, would be more suited for the "Library improvements" section